src/main/java/lotto/domain/LottoStatistics.java

```diff
@@ -0,0 +1,60 @@
+package lotto.domain;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoStatistics {
+
+    private Map<LottoRank,Integer> lottoStaticResult;
+
+    public LottoStatistics() {
+        lottoStaticResult = new HashMap<>();
+        init();
+    }
+
+    private void init() {
+        for (LottoRank lottoRank : LottoRank.values()) {
+            lottoStaticResult.put(lottoRank,0);
+        }
+    }
+
+    public void put(LottoRank rank) {
+        if(!LottoRank.isZero(rank)) {
+            lottoStaticResult.put(rank,lottoStaticResult.get(rank)+1);
```

Comment:
```suggestion
            lottoStaticResult.put(rank, lottoStaticResult.get(rank) + 1);
```
이러한 컨벤션 규칙은 ide 에서 제공하는 reformat 기능을 활용하시면 일괄적으로 변경이 가능합니다 :)

Reply:
꿀팁 감사합니당